apps/src/sites/studio/pages/components/StageCard.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+import { borderRadius, levelTokenMargin } from './constants';
+import OrderControls from './OrderControls';
+import LevelToken from './LevelToken';
+
+const styles = {
+  checkbox: {
+    margin: '0 0 0 7px'
+  },
+  stageCard: {
+    fontSize: 18,
+    background: 'white',
+    border: '1px solid #ccc',
+    borderRadius: borderRadius,
+    padding: 20,
+    margin: 10
+  },
+  stageCardHeader: {
+    color: '#5b6770',
+    marginBottom: 15
+  },
+  stageLockable: {
+    fontSize: 13,
+    marginTop: 3
+  },
+  addLevel: {
+    fontSize: 14,
+    background: '#eee',
+    border: '1px solid #ddd',
+    boxShadow: 'inset 0 1px 0 0 rgba(255, 255, 255, 0.8)',
+    margin: '5px 0'
+  }
+};
+
+const StageCard = React.createClass({
+  propTypes: {
+    reorderLevel: React.PropTypes.func.isRequired,
+    addLevel: React.PropTypes.func.isRequired,
+    stagesCount: React.PropTypes.number.isRequired,
+    stage: React.PropTypes.object.isRequired
+  },
+
+  getInitialState() {
+    return {currentPositions: []};
```

Comment:
This might be personal preference, but I like to provide default values for any state I might set in getInitialState so that there's one place I can go to look and see what state this component uses